config/crd/bases/delivery.ocm.software_componentversions.yaml

```diff
@@ -126,8 +127,7 @@ spec:
                   type: object
```

Comment:
Object is not sufficient, need a structure so that it doesn't show 'object' in CRD readers like https://crds.cronohub.org/

Reply:
Object is not  for documentation purposes and there is a technical requirement as to why this type is used.